src/app/functions/server/getUserBreaches.ts

```diff
@@ -96,7 +97,12 @@ export async function getUserBreaches({
 export async function getSubscriberBreaches(
   user: Session["user"],
 ): Promise<SubscriberBreach[]> {
+  // FIXME this does not always return a subscriber https://mozilla-hub.atlassian.net/browse/MNTOR-2936
   const subscriber = await getSubscriberByEmail(user.email);
```

Comment:
Do we know if `user.email` is always present and not the problem?

Might as well log that if not

Reply:
oh sorry, i misread..  that makes sense